index.html

```diff
@@ -169,6 +169,33 @@ <h2>
         <dfn>MIDI message</dfn>, <dfn>System Real Time</dfn> and
         <dfn>System Exclusive</dfn> are defined in [[MIDI]].
       </p>
+      <div class="note">
+        The definition of a <dfn>valid MIDI message</dfn> should be
```

Comment:
I think this should be "[the definition of a valid message] is defined in" (not "should be inferred by").  It's defined in the spec.

Reply:
The MIDI 1.0 spec doesn't explicitly define a valid MIDI message, although it does refer to a "valid Status byte".  Maybe it's not necessary to have an explicit definition?  How does the wording look now?

Reply:
LGTM.  It's true they don't use the term "valid MIDI message", but the structure of a message is clearly defined starting in page 4.  New wording is good.

BTW, I noted the MIDI specref pointer is out of date - the MMA moved the spec on their site and didn't up date it - and I filed a pull request in specref to update it.